lib/app/router.dart

```diff
@@ -3,18 +3,35 @@ import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:go_router/go_router.dart';
 import 'package:gruene_app/app/auth/bloc/auth_bloc.dart';
 import 'package:gruene_app/app/constants/routes.dart';
+import 'package:gruene_app/app/widgets/bottom_navigation.dart';
+
+final GlobalKey<NavigatorState> _rootNavigatorKey = GlobalKey<NavigatorState>(debugLabel: 'root');
 
 GoRouter createAppRouter(BuildContext context) {
   return GoRouter(
+    navigatorKey: _rootNavigatorKey,
     initialLocation: Routes.news.path,
     routes: [
-      Routes.news,
-      Routes.campaigns,
-      Routes.profiles,
-      Routes.mfa,
-      Routes.tools,
-      Routes.settings,
       Routes.login,
+      Routes.mfaLogin,
```

Comment:
I don't really like this solution of having two routes for mfa (one if logged in, one if logged out) but I did not find a better or even working solution.